src/layouts/Default.vue

```diff
@@ -52,6 +52,15 @@ export default {
     this.getCredentials().then(() => {
       this.updateAuth();
     });
+
+    if (window.ipc) {
+      window.ipc.on("CREDENTIAL_CHANGE", () => {
+        this.getCredentials().then(() => {
+          this.updateAuth();
+        });
+        this.$forceUpdate();
+      });
+    }
```

Comment:
Why did you duplicate the `window.ipc.on` for credential changes? It already exists in `mounted()`?

Reply:
Didn't see it in the mounted(). I removed it in dev, retested and it looks good.